### ISSUE TITLE
Question: "archivebox: command not found"

### ISSUE THREAD

**Issue by Slusho64** [pirate/ArchiveBox#462]
Please bear with me as I'm a novice Linux user. I followed the quickstart guide, cloning the git repo and running the setup script, which all went well. however, when I try to run `archivebox add < ~/bookmarks.html` (I put my file in my home/[username] director), it says "archivebox: command not found". Is this a PATH issue? How can I fix this?

I tried on Ubuntu and WSL2 Ubuntu (not sure if this should work on WSL or if you need to use Docker on that or what all the limitations are of that system).

Also, on the Configuration page on this project, there's a link "Creating a Config File" that seems to lead to a nonexistent section of that same page, so I'm not sure how to configure archives.

**Comment by cdvv7788** [comment]
@Slusho64 what instructions did you follow? did you run `pip install archivebox`? Can you run `pip freeze` and check that archivebox is in the list?

**Comment by Slusho64** [comment]
I used the Quickstart guide, cloning the git and then running the setup script for the dependencies. It didn't say to use pip. I tried pip after reading your comment but I couldn't get it to work for hours until I found a comment that made me realize it's not pip but pip3 that must be run. Now I have ArchiveBox installed and running. However, when I add or update, it gives errors that it could not make pdf, screenshot, or DOM backups (I do not understand what wget, warc, and dom are and which of them are necessary for an html backup, and I haven't been able to find a clear explanation in the documentation or even via web searches) both in Linux and WSL. On Linux, the backups are still made but on WSL, the first error causes the whole thing to stop running. So I am able to run it on Linux, albeit without those backup features, and the backups that it produces are sometimes excellent and sometimes very screwed up.

**Comment by pirate** [comment]
WSL is not officially supported, I recommend using Linux or Docker only.

**Comment by Slusho64** [comment]
Thanks for the info on WSL. Any idea why screenshot, PDF, and sometimes DOM don't work on my proper Ubuntu install either? Do I even want DOM (again, I can't find what it is)?

**Comment by cdvv7788** [comment]
What error is it raising in those cases? The easiest way to use `archivebox` is docker...this way you avoid missing dependencies.

**Comment by Slusho64** [comment]
Alright I tried Docker and it wouldn't work in WSL or Ubuntu proper (it wouldn't process anything in the bookmarks file) so I just went back to the previous way I got it working in Ubuntu and ran it, errors be damned. I did run into it hanging on the DOM portion of a site that was no longer online so I edited it out of my HTML file. I've spent as much time as I'd like on this and I got a reasonably satisfactory result. Thanks for the help.
